web/Web/web/review/templates/review/review.html

```diff
@@ -0,0 +1,293 @@
+{% extends "base.html" %}
+{% load static %}
+{% block pagetitle %}Review{% endblock %}
+{% block navbar-form %}
+<form id="searchContentForm"
+    class="form-inline my-2 my-lg-0 order-sm-1 order-md-3 align-self-center align-self-stretch">
+    <div class="input-group">
+        <input id="search_content" autocomplete="off" type="search"
+            class="form-control search-input bg-light text-dark no-border shadow-none" placeholder="Highlight">
+
+        <div class="input-group-append">
+            <button class="btn btn-sm btn-light" data-search="next" id="searchNext" type="button">&darr;</button>
+            <button class="btn btn-sm btn-light" data-search="prev" type="button">&uarr;</button>
+            <!--<button class="btn btn-sm btn-light" data-search="clear" type="button">&times;</button>-->
+        </div>
+    </div>
+</form>
+{% endblock %}
+
+
+{% block navbar-center %}
+<div class=" nav navbar-nav mx-auto justify-content-center center-text pl-2 pr-2">
+    <form class="form-inline">
+        <div class="btn-group" role="group">
+            <a href="#" class="btn btn-sm btn-outline-secondary d-flex align-items-center docViewDocNonRelButton">Not
+                {% if main_judging_criteria_name %}{{ main_judging_criteria_name|truncatechars:3 }}{% else %}Rel{% endif %}</a>
+            <a href="#"
+                class="btn btn-sm btn-outline-secondary d-flex align-items-center docViewDocRelButton">{% if main_judging_criteria_name %}{{ main_judging_criteria_name|truncatechars:3 }}{% else %}Rel{% endif %}</a>
+            <a href="#" class="btn btn-sm btn-outline-secondary d-flex align-items-center docViewDocHRelButton">Highly
+                {% if main_judging_criteria_name %}{{ main_judging_criteria_name|truncatechars:3 }}{% else %}Rel{% endif %}</a>
+        </div>
+    </form>
+</div>
+{% endblock %}
+
+
+{% block main %}
+
+
+<div class="mb-4">
+
+    <div class="row">
+        <div class="col-12 col-md-8">
+
+            <nav class="nav nav-pills">
+                <div class="d-flex px-3 py-2">
+                    <span class="pr-2 py-1 align-self-end small align-self-center text-muted border-right unselectable"
+                        unselectable="on">Document viewer settings</span>
+                    <a class="nav-link small ml-2 px-2 py-1 align-self-end active" href="#"><i
+                            class="fas fa-highlighter"></i></a>
+                    <a class="nav-link small ml-2 px-2 py-1" data-toggle="modal" data-target="#keyboardShortcutsModal"
+                        href="#"><i class="fas fa-keyboard"></i></a>
+                    <a class="nav-link small ml-2 px-2 py-1 disabled" href="#" tabindex="-1"><i
+                            class="fas fa-tag"></i></a>
+                </div>
+                <div>
```

Comment:
Minor edit but it will make things look better. 
Put the two buttons in a btn-group class. 
```
<div class="btn-group" role="group" aria-label="Basic example">
  <button type="button" class="btn btn-sm btn-outline-secondary d-flex align-items-center docViewPreviousDocButton">Prev</button>
  <button type="button" class="btn btn-sm btn-outline-secondary d-flex align-items-center docViewNextDocButton">Next</button>
</div>
```

Reply:
Done